src/components/homepage/SectionTwo.jsx

```diff
@@ -139,15 +142,81 @@ const SectionTwo = () => {
                     ))}
                 </div>
             </div>
+            {/* events images section */}
+            <div className='flex md:flex-row flex-col px-4 gap-3 md:px-8 py-4'>
+                <div className='hidden md:flex flex-col gap-y-3'>
+                    <div className='h-[40%] rounded-md overflow-hidden shadow-md'>
+                        <Image
+                            className='w-full h-full transition duration-300 hover:scale-110'
+                            src='/globalGoalsLogos/refugies.png'
+                            alt=''
+                            width={800}
+                            height={800}
+                        />
+                    </div>
+                    <div className='h-[60%] rounded-md overflow-hidden shadow-md'>
+                        <Image
+                            className='w-full h-full transition duration-300 hover:scale-110'
+                            src='/globalGoalsLogos/helpOlds.png'
+                            alt=''
+                            width={800}
+                            height={800}
+                        />
+                    </div>
+                </div>
+                <div className='hidden md:flex flex-col gap-y-3'>
+                    <div className='h-[55%] rounded-md overflow-hidden shadow-md'>
+                        <Image
+                            className='w-full h-full transition duration-300 hover:scale-110'
+                            src='/globalGoalsLogos/construct.png'
+                            alt=''
+                            width={800}
+                            height={800}
+                        />
+                    </div>
+                    <div className='h-[45%] rounded-md overflow-hidden shadow-md'>
+                        <Image
+                            className='w-full h-full transition duration-300 hover:scale-110'
+                            src='/globalGoalsLogos/donate.png'
+                            alt=''
+                            width={800}
+                            height={800}
+                        />
+                    </div>
+                </div>
+            </div>
+            <div className='md:hidden px-2'>
+                <Carousel
+                    showArrows={false}
+                    infiniteLoop={true}
+                    autoPlay={true}
+                    interval={4000}
+                    showStatus={false}
+                    showIndicators={false}
+                >
+                    <div>
```

Comment:
a better alternative would be to use the Image component, same As for Line:  198, 201, 204, 207